apps/etterlatte-behandling/src/main/kotlin/behandling/revurdering/ManuellRevurderingService.kt

```diff
@@ -79,14 +78,6 @@ class ManuellRevurderingService(
             begrunnelse = begrunnelse,
             fritekstAarsak = fritekstAarsak,
             saksbehandler = saksbehandler,
-            opphoerFraOgMed =
```

Comment:
Litt usikker på denne, men trenger vi egentlig logikk for dette? Er det ikke naturlig at opphør fom følger med i revurderingen, og kan endres / fjernes av saksbehandler i revurderingsoversikten?